code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2172,3 +2182,14 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjust_fire_stacks(1)
 		M.IgniteMob()
 	..()
+
+/datum/reagent/consumable/ethanol/painkiller
+	name = "Painkiller"
```

Comment:
I kind of feel like a drink named "painkiller" should reduce your slowdown from damage or something, y'know?

speaking of which, has ignore_slowdown() been unborked yet?

Reply:
If you read the flavor text, it's a bit of a joke in itself that the drink doesn't do anything for your physical pain. I'd be willing to give it a bigger mood buff or perhaps a very small tweak to damage/slowdown. I'll investigate and see what would be reasonable without making the drink _too_ helpful.

Reply:
Per dr-bee's suggestion, I opted to have Painkiller hide damage on the health HUD, while not healing at all.